prow/manifests/overlays/metal3/config.yaml

```diff
@@ -161,7 +161,7 @@ branch-protection:
                 contexts: ["test-ubuntu-integration-release-1-3"]
             release-1.4:
               required_status_checks:
-                contexts: ["test-ubuntu-integration-release-1-4"]
+                contexts: ["test-ubuntu-e2e-integration-release-1-4"]
```

Comment:
same for line 177 no more ansible tests on releases

Reply:
thats for dev-env, we should only do it for capm3 and ipam now I understood ? And this way we keep atleast the ansible scripts also being tested ?

Reply:
My idea is to no loner support ansible tests on for the new releases but if you want to keep supporting ansible integrations on dev-env even when they are deprecated on capm3 we can keep it but we'll end up duplicating the same test with different implementations